src/binary/requests/completionPostProcess.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  AutocompleteParams,
+  AutocompleteResult,
+  CompletionKind,
+} from "./requests";
+
+export default function postprocess(
+  request: AutocompleteParams,
+  result: AutocompleteResult,
+  tabSize: number
+): void {
+  const resultsSubset = result.results.filter(
+    (entry) => entry.completion_kind === CompletionKind.Snippet
+  );
+  if (resultsSubset.length === 0) return;
+
+  resultsSubset.forEach((entry, index) => {
+    resultsSubset[index].new_prefix = entry.new_prefix.replace(
+      /\t/g,
+      " ".repeat(tabSize)
```

Comment:
What if using tabs and not spaces for ident?

Reply:
We're always returning indentation in spaces in multiline suggestions, assuming the ides know how to handle it - im just normalizing the suggestion to this standard, because we're working now with some server that return tabs from time to time